lib/hardware/modules/div/iob_div_subshift_frac/iob_div_subshift_frac.py

```diff
@@ -13,22 +22,196 @@ def setup(py_params_dict):
-        "blocks": [
+        "wires": [
             {
-                "core_name": "iob_reg",
-                "instance_name": "iob_reg_inst",
+                "name": "divisor_reg",
+                "descr": "divisor_reg wire",
+                "signals": [
+                    {"name": "divisor_reg", "width": "DATA_W"},
+                ],
+            },
+            {
+                "name": "quotient_int",
+                "descr": "quotient_int wire",
+                "signals": [
+                    {"name": "quotient_int", "width": "DATA_W"},
+                ],
+            },
+            {
+                "name": "incr",
+                "descr": "incr wire",
+                "signals": [
+                    {"name": "incr", "width": 1},
+                ],
             },
             {
-                "core_name": "iob_reg_e",
-                "instance_name": "iob_reg_e_inst",
+                "name": "res_acc",
+                "descr": "res_acc wire",
+                "signals": [
+                    {"name": "res_acc", "width": "DATA_W+1"},
+                ],
+            },
+            {
+                "name": "pc",
+                "descr": "pc wire",
+                "signals": [
+                    {"name": "pc", "width": 2},
+                ],
+            },
+            {
+                "name": "div_frac",
+                "descr": "Division interface",
+                "signals": [
+                    {
+                        "name": "dividend",
+                    },
+                    {
+                        "name": "divisor",
+                    },
+                    {
+                        "name": "quotient_int",
+                    },
+                    {
+                        "name": "remainder",
+                    },
+                ],
+            },
+        ],
+        "blocks": [
+            {
+                "core_name": "iob_reg",
+                "instance_name": "divisor_reg0",
+                "parameters": {
+                    "DATA_W": "DATA_W",
+                    "RST_VAL": "1'b0",
+                },
+                "connect": {
+                    "clk_en_rst": "clk_en_rst",
+                    "data_i": "divisor_i",
+                    "data_o": "divisor_reg",
+                },
             },
             {
                 "core_name": "iob_div_subshift",
-                "instance_name": "iob_div_subshift_inst",
+                "instance_name": "div_subshift0",
+                "parameters": {
+                    "DATA_W": "DATA_W",
+                },
+                "connect": {
+                    "clk_en_rst": "clk_en_rst",
+                    "div": "div_frac",
+                },
             },
         ],
+        "combs": [
+            {
+                "verilog_code": """
+            incr        = 1'b0;
+            res_acc_nxt = res_acc + remainder_o;
+            res_acc_en  = 1'b0;
+            pc_nxt      = pc + 1'b1;
```

Comment:
```suggestion
```
I think py2hwsw fsms already increment pc_nxt by default, so we don't need this line.